src/modules/launcher/Wox.Infrastructure/Helper.cs

```diff
@@ -113,6 +113,7 @@ public static Process OpenInConsole(string path)
             {
                 WorkingDirectory = path,
                 FileName = "cmd.exe",
+                Arguments = $"/C wt.exe -d {path} 2> NUL >NUL || cmd.exe",
```

Comment:
Can we get problems with opening cmd/powershell within cmd (e. g. incorrect environment vars)? I personally worry with this way of invocation.

Reply:
The working directory for cmd is still set the same way as before, so I don't think we will have issues here.

I will test this and take a look/resolve/reply to the other comments in this PR this weekend.

Thanks for taking a look at this!

Reply:
An other reason to call cmd/wt directly is that et (called directly) is working also when cmd ist blocked/broken.